preinstall.js

```diff
@@ -25,9 +25,12 @@ if (!conf) {
   console.error(`Platform ${process.platform} is not supported`)
   process.exit(1)
 }
-const endpointURL = `https://s3-eu-west-1.amazonaws.com/ledger-lib-ledger-core/${libcoreVersion}/${
-  conf.dir
-}`
+
+const dir = process.platform === "linux" &&
+Number(process.version.match(/^v(\d+\.\d+)/)[1]) >= 10  ?
```

Comment:
I don't think we should do it like this. The runtime install process is not necessarily the one used by the app (case of electron, you can install with node 12 , yet use electron1)

Can you explain why we would not do it for all node versions?

Reply:
@gre we checked and it will be fine: 

- Building bindings with `node-8` and `yalc`ed to minimal project on `electron-7`: electron is always going through the `preinstall.js` which is exactly the behaviour we want.

Only thing to adapt is pre-builds (talked to @IAmMorrow ), but since you want to fully move to `node-12` then you should not have a lot to do !

Reply:
I also created a ticket from our side related to `node8` depreciation: https://ledgerhq.atlassian.net/browse/LLC-533 
Just to get rid of CI step doing so and also published artifacts (we will still be theoretically supporting `node8`)